hello_world/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "hello-world"
+name = "hello_world"
```

Comment:
Good. Follows [naming conventions](https://doc.rust-lang.org/1.0.0/style/style/naming/README.html).

Reply:
Wasn't aware of the naming convention. While compiling realized that rustc is automatically converting "-" to "_".